src/components/shared/MarkerContextMenu.js

```diff
@@ -94,14 +94,20 @@ class MarkerContextMenu extends PureComponent<Props> {
       ? previewSelection.selectionStart
       : committedRange.start;
 
+    let selectionEnd = rightClickedMarker.end || rightClickedMarker.start;
+
+    if (selectionEnd === selectionStart) {
+      // For InstantMarkers, or Interval markers with 0 duration, add an arbitrarily
+      // small bit of time at the end to make sure the selected marker doesn't disappear
+      // from view.
+      selectionEnd += 0.0001;
```

Comment:
Do we still need that special handling after `getStartEndRangeForMarker`? 
Hm, I guess we are not using it here. Is it not possible?

Reply:
I'm a bit nervous about setting a 0 length selection, as this will cause the range selections to have divide by 0 errors. I'd prefer to keep this here. I'm not 100% on the meaning here of it now. I don't think `getStartEndRangeForMarker` is running here.